coriolis_openstack_utils/actions.py

```diff
@@ -753,3 +755,157 @@ def execute_operations(self):
         # done_ids = [migr["existing_migration_id"]
         #             for migr in self._completed_migrations]
         return migrations
+
+
+class SecurityGroupCreationAction(Action):
+    action_type = ACTION_TYPE_CHECK_CREATE_SECGROUP
+    NEW_SECGROUP_DESCRIPTION = (
+        "Created by the Coriolis OpenStack utilities for source security group"
+        " '%s'.")
+
+    def __init__(self, source_client, destination_client, action_payload):
+        """
+        param source_client: source client needed for fetching source
+        secgroup info
+        param destination_client: destination client needed for secgroup
+        creation
+        param action_payload: dict(): payload (params) for the action
+        must contain 'src_tenant_id' or 'src_project_id'
+                     'dest_tenant_id' / 'dest_tenant_id'
+                     'source_name'
+        """
+        self._source_client = source_client
+        self._destination_client = destination_client
+        self.payload = action_payload
+        self.subactions = []
+
+    @property
+    def secgroup_name_format(self):
+        return CONF.destination.new_secgroup_name_format
+
+    def get_source_tenant_id(self):
```

Comment:
Considering you only ever set `src_tenant_id` and `dest_tenant_id` when creating the payload these action classes get instantiated with, both this and the below `get_destination_tenant_id` are redundant.
https://github.com/cloudbase/coriolis-openstack-utils/pull/9/files#diff-c1504cba7bc0f50d3e4c3aed304956d2R80